pkg/schema/schema_test.go

```diff
@@ -13,13 +13,13 @@ import (
 	"golang.org/x/tools/txtar"
 )
 
-var CasesDir = filepath.Join("load", "testdata", "artifacts", "dashboards", "trimdefault")
+var CasesDir = filepath.Join("load", "testdata", "trimdefault")
```

Comment:
nit: `testdata/trimdefault`, not `load/testdata/trimdefault`. Absent a good reason not to, tests should look in a `testdata` dir that's sibling to the test file itself.